queue_processors/autoreduction_processor/README.md

```diff
@@ -1,10 +1,26 @@
-## AutoreductionProcessor
+## autoreduction_processor module
 
-These scripts handle the messages added to the following queues:
-* `/ReductionPending`
-* `/ReductionStarted`
-* `/RedcutionComplete`
-* `/ReductionError`
+This Daemon handles the management of processes and the processes themselves that perform the reduction of the data.
 
-These scripts focus more on the reduction of the data and are used to control
-the reduction jobs that get pushed to the reduction software (at ISIS this is MANTID).
\ No newline at end of file
+The `autoreduction_processor.py` is responsible for the management of redcution processes. It is run as a linux Daemon using `autoreduction_processor_daemon.py` as a control script to start/stop/restart the daemon process. As it is a daemon it can only be run in linux environments. The daemon is normally started with the `queue_processor.py` daemon via the `../restart.sh` script. 
+
+The `autoreduction_processor` itself is an ActiveMQ consumer that is implemented using the stomp library. It consumes messages from the `/ReductionPending` queue and starts a new python instance running `post_process_admin.py` for each valid message it reads from the queue.
+
+`post_proces_admin.py` is the script that is responsible for the actual reduction of data. The *main* responsiblities of the script are:
+* Sending a message to `/ReductionStarted` to update the status of the job in the database (from *queued* to *started*)
+* Fetching the reduction script
+* Setting up reduction environment
+  * Creating a temporary directory for resulting reduced data
+  * Redirecting logging information
+* Starting the reduction process in external software (such as `Mantid`)
+* Handling what happens after the reduction process has ended:
+  * If the job was successful and no errors occured
+    * Copy reduced data to its long term storage directory (at ISIS this is on CEPH)
+    * Send a message to `/RedcutionComplete`
+  * If the job was unsuccessful (Any errors were raised, exceptions thrown)
+    * Send a message to `/ReductionError`
+  * The job **SHOULD** not be completed, i.e. it should be skipped. E.g. the run does not require reduction, it is found running in unknown mode of operation, the reduction script does not support the type of data it was given (such as event mode on EnginX) etc.
```

Comment:
Hi Elliot, should `The job **SHOULD** not be completed, i.e. it should be skipped` be changed to `The job should be skipped`. Since it was not obvious to me why you emphasized **SHOULD**.

Also, could you also double the slight rewording of this bullet point? Thanks

Reply:
Changes made